SDWebImage/Core/SDAnimatedImageView.h

```diff
@@ -20,6 +20,13 @@
  */
 @interface SDAnimatedImageView : UIImageView
 
+/**
+ If the image has more than one frame, set this value to `YES` will automatically
+ play/stop the animation when the view become visible/invisible.
+ 
+ The default value is `YES`.
+ */
+@property (nonatomic) BOOL autoPlayAnimatedImage;
```

Comment:
Move this property below the `resetFrameIndexWhenStopped` is better.

The first thing user care about is the `currentFrame` mostly, so which should be listed in the Public Interface at first line.

Reply:
Adjustments have been made